x-pack/scripts/functional_tests.js

```diff
@@ -34,4 +34,5 @@ require('@kbn/test').runTestsCli([
   require.resolve('../test/ui_capabilities/security_only/config'),
   require.resolve('../test/ui_capabilities/spaces_only/config'),
   require.resolve('../test/upgrade_assistant_integration/config'),
+  require.resolve('../test/integrations_manager_api_integration/config.ts'),
```

Comment:
How about

```suggestion
  require.resolve('../test/integrations_manager_api_integration/config'),
```
to match the other entries

Reply:
Looking further up I noticed that the `.ts` versions include their file ending, and I didn't want to spend a whole CI cycle trying if it works without it.